designer/server/src/main/scala/pl/touk/nussknacker/ui/process/ProcessService.scala

```diff
@@ -91,6 +91,7 @@ object ProcessService {
     case class ValidateAndResolve(includeValidationNodeResults: Boolean = true) extends ValidationMode
   }
 
+  @JsonCodec case class GetLastlyDeployedVersionId(versionId: Option[Long])
```

Comment:
Maybe we could remove verb `Get` from class name ? WDYT?

Reply:
Yeahh, that sounds much better :)